internal/git/git.go

```diff
@@ -559,88 +571,126 @@ func (g *Git) getOwnerAndRepo(githubRepoLocation string) (string, string) {
 }
 
 func (g *Git) CreateOrUpdatePR(info PRInfo) (*github.PullRequest, error) {
+	logging.Debug("Starting: Create or Update PR")
+	labelTypes := g.UpsertLabelTypes(context.Background())
 	var changelog string
 	var err error
-
-	labelTypes := g.UpsertLabelTypes(context.Background())
-
+	body := ""
 	var previousGenVersions []string
-
+	var title string
 	if info.PreviousGenVersion != "" {
 		previousGenVersions = strings.Split(info.PreviousGenVersion, ";")
 	}
+	packageName := ""
+	languageForPr := ""
+
+	if info.GenInfo != nil && info.ReleaseInfo != nil {
+		for lang, langGenerationInfo := range info.GenInfo.Languages {
+			if _, exists := info.ReleaseInfo.LanguagesGenerated[lang]; !exists || langGenerationInfo.PackageName == "" {
+				continue
+			}
+			languageForPr = lang
```

Comment:
Hmmm, if multiple languages present in PR we shouldn't flatten this information in the PR title.